### PR TITLE
RDI-1892: Uniform examples structure

### PR DESCRIPTION
**Pull request by cfoch**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  hooks:
+  - id: clang-format
+    types_or: [c++, c, cuda]
+-   repo: https://github.com/pre-commit/mirrors-prettier
+    rev: 'v3.1.0'
+    hooks:
+    - id: prettier
+      additional_dependencies:
+        - prettier@3.1.0
+      types_or: [html, javascript]
```
Comment (rgonzalezfluendo) [inline .pre-commit-config.yaml:19]: identation issue

Comment (cfoch) [reply .pre-commit-config.yaml:19]: What do you mean? No errors here according pre-commit check.

Comment (rgonzalezfluendo) [reply .pre-commit-config.yaml:19]: compare

```
repos:
- repo: https://github.com/pre-commit/pre-commit-hooks
  rev: v2.3.0
  hooks:
  - id: check-yaml
  - id: end-of-file-fixer
  - id: trailing-whitespace
- repo: https://github.com/pre-commit/mirrors-clang-format
  rev: v14.0.6
  hooks:
  - id: clang-format
    types_or: [c++, c, cuda]
-   repo: https://github.com/pre-commit/mirrors-prettier
    rev: 'v3.1.0'
    hooks:
    - id: prettier
      additional_dependencies:
        - prettier@3.1.0
      types_or: [html, javascript]
- repo: https://github.com/astral-sh/ruff-pre-commit
  # Ruff version.
  rev: v0.1.8
  hooks:
    # Run the linter.
    - id: ruff
      args: [ --fix, --show-fixes ]
    # Run the formatter.
    - id: ruff-format
      files: '.*\.(recipe|py)'
```

vs

```
---
repos:
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v2.3.0
    hooks:
      - id: check-yaml
      - id: end-of-file-fixer
      - id: trailing-whitespace
  - repo: https://github.com/pre-commit/mirrors-clang-format
    rev: v14.0.6
    hooks:
      - id: clang-format
        types_or:
          - c++
          - c
          - cuda
  - repo: https://github.com/pre-commit/mirrors-prettier
    rev: v3.1.0
    hooks:
      - id: prettier
        additional_dependencies:
          - prettier@3.1.0
        types_or:
          - html
          - javascript
  - repo: https://github.com/astral-sh/ruff-pre-commit
    rev: v0.1.8
    hooks:
      - id: ruff
        args:
          - --fix
          - --show-fixes
      - id: ruff-format
        files: .*\.(recipe|py)
```

**Comment by rgonzalezfluendo** [comment]
`README.md` needs to be updated.

```diff
diff --git a/README.md b/README.md
index 01a9ae2..e8657dc 100644
--- a/README.md
+++ b/README.md
@@ -100,15 +100,15 @@ To build an environment, you can use [steps from CI](.github/workflows/gst.wasm-
 ``
 git clone https://github.com/fluendo/cerbero cerbero-src -b gst.wasm --depth=1
 ./cerbero-src/cerbero-uninstalled -c build/gst.wasm.cbc bootstrap
-./cerbero-src/cerbero-uninstalled -c build/gst.wasm.cbc build-deps gst.wasm
+./cerbero-src/cerbero-uninstalled -c build/gst.wasm.cbc build gst.wasm
 ``
 
-As a result, in the local directory `build/build/gst.wasm_linux_web_wasm32/dist/web_wasm32`
+As a result, in the local directory `build/gst.wasm_web_wasm32/dist/web_wasm32`
 you've got all the needed packages with corresponding `*.pc` files, which could be used
 to build your project based on the WASM version of GStreamer and its dependencies:
 
 ``
-$ ls build/build/gst.wasm_linux_web_wasm32/dist/web_wasm32/
+$ ls build/gst.wasm_web_wasm32/dist/web_wasm32/
 bin  include  lib  share
 ``

```

**Review by rgonzalezfluendo — APPROVED** [review]

**Pull request merged**